frontend/src/scenes/billing/PlanComparison.tsx

```diff
@@ -288,6 +288,7 @@ export const PlanComparison = ({
                 )}
                 {includeAddons &&
                     product.addons?.map((addon) => {
+                        // TODO: integrated_identities addon will show up here when we add a price plan. Make sure this can handle it.
```

Comment:
```suggestion
                        // TODO: integrated_persons addon will show up here when we add a price plan. Make sure this can handle it.
```